crates/net/network/src/import.rs

```diff
@@ -46,6 +58,11 @@ pub enum BlockValidation<B> {
         /// validated block.
         block: NewBlockMessage<B>,
     },
+    /// Valid block hash announcement
+    ValidBlockHashes {
+        /// Block hashes that were validated
+        hashes: NewBlockHashes,
+    },
```

Comment:
this we should leave unchanged and instead only introduce the new enum NewBlockEvent